client/src/main/scala/it/cwmp/client/view/game/GameFX.scala

```diff
@@ -1,43 +1,64 @@
 package it.cwmp.client.view.game
 
 import java.awt.Color
-import it.cwmp.client.model.game.Point
+
+import it.cwmp.client.model.game.{Point, World}
 import it.cwmp.client.view.game.model._
-import javafx.application.Application
+import javafx.application.{Application, Platform}
+import javafx.embed.swing.JFXPanel
 import javafx.scene.canvas.Canvas
 import javafx.scene.{Group, Scene}
 import javafx.stage.Stage
 
+
+object GameFX {
+  def apply(): GameFX = new GameFX()
+}
+
```

Comment:
Al posto di fare un companion solo per la apply, si può mettere direttamente la classe come "case class", avrà l'apply gratis ;)

Reply:
In realtà ci avevo messo altri metodi che poi ho tolto, ora rifattorizzo! Grazie